tests/functional/a/alternative/alternative_union_syntax_error.py

```diff
@@ -120,3 +120,11 @@ class WithHorrible(metaclass=HorribleMetaclass):
     pass
 
 class_list = [WithHorrible | DefaultMetaclass]
+
+class SecondLevelMetaclass(ForwardMetaclass):
+    pass
+
+class WithSecondLevel(metaclass=SecondLevelMetaclass):
+    pass
+
+class_list_with_second_level = [WithSecondLevel | DefaultMetaclass]
```

Comment:
```suggestion
class_list_with_second_level = [1 | "DefaultMetaclass"]
```
To cover the last line of ``_recursive_search_for_classdef_type`` ?